lte/gateway/c/core/oai/tasks/amf/nas_proc.cpp

```diff
@@ -680,8 +682,8 @@ int amf_decrypt_imsi_info_answer(itti_amf_decrypted_imsi_info_ans_t* aia) {
    * Execute the requested new UE registration procedure
    * This will initiate identity req in DL.
    */
-  rc = amf_proc_registration_request(aia->ue_id, is_amf_ctx_new, params);
-  OAILOG_FUNC_RETURN(LOG_NAS_AMF, rc);
+  amf_proc_registration_request(aia->ue_id, is_amf_ctx_new, params);
+  return imsi64;
```

Comment:
Replace the return statement with, OAILOG_FUNC_RETURN and return the imsi64

Reply:
addressed